curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da97c987a514959ada414.md

```diff
@@ -0,0 +1,189 @@
+---
+id: 641da97c987a514959ada414
+title: Step 22
+challengeType: 0
+dashedName: step-22
+---
+
+# --description--
+
+Now call the `displayAuthors` function with a portion of the author data just like you did before.
+
+If you click the `Load More Authors` button after calling the function, it won't work. That's because you still have to add the `click` event listener to the button. You'll do that next.
+
+# --hints--
```

Comment:
Right now, I am able to pass without writing any code

<img width="727" alt="Screen Shot 2023-04-22 at 4 24 59 PM" src="https://user-images.githubusercontent.com/67210629/233811697-7426538b-e917-4126-ac0c-361e2b9f33fc.png">

Reply:
Thanks for checking that, Jessica. I've fixed it. Check it out.